generators/app/templates/_.eslintrc

```diff
@@ -1,54 +1,85 @@
 {
-	"env": {
-		"browser": true,
-		"node": true
-	},
-	"parserOptions": {
-		"ecmaVersion": 2019
-	},
-	"globals": {
-		"sap": true,
-		"jQuery": true
-	},
-	"rules": {
-		"block-scoped-var": 1,
-		"keyword-spacing": 2,
-		"space-unary-ops": 2,
-		"camelcase": 1,
-		"consistent-return": 1,
-		"no-warning-comments": 1,
-		"default-case": 1,
-		"no-console": 2,
-		"no-unused-vars": 2,
-		"no-debugger": 2,
-		"no-trailing-spaces": [
-			2,
-			{
-				"ignoreComments": true
-			}
-		],
-		"semi": [
-			1,
-			"always"
-		],
-		"quotes": [
-			1,
-			"double"
-		],
-		"key-spacing": [
-			1,
-			{
-				"beforeColon": false
-			}
-		],
-		"comma-spacing": [
-			1,
-			{
-				"before": false,
-				"after": true
-			}
-		],
-		"no-shadow": 2,
-		"no-irregular-whitespace": 2
-	}
-}
\ No newline at end of file
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true,
+        "jquery": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 2020
+    },
+    "globals": {
+        "sap": true,
+        "jQuery": true
+    },
+    "rules": {
+        "array-callback-return": "error",
+        "arrow-spacing": "error",
+        "brace-style": "error",
+        "camelcase": ["warn"],
+        "comma-spacing": [
+            "warn",
+            {
+                "before": false,
+                "after": true
+            }
+        ],
+        "comma-style": "error",
+        "consistent-return": "warn",
+        "default-case": "warn",
+        "default-param-last": "error",
+        "eqeqeq": "error",
+        "id-length": "off",
+        "key-spacing": [
+            "warn",
+            {
+                "beforeColon": false
+            }
+        ],
+        "keyword-spacing": "error",
+        "max-lines": [
+            "warn",
+            {
+                "max": 500,
+                "skipBlankLines": true,
+                "skipComments": true
+            }
+        ],
+        "new-cap": "warn",
+        "no-array-constructor": "error",
+        "no-confusing-arrow": "error",
+        "no-console": "error",
+        "no-debugger": "error",
+        "no-eval": "error",
+        "no-else-return": "error",
+        "no-loop-func": "error",
+        "no-irregular-whitespace": "error",
+        "no-mixed-operators": "error",
+        "no-multi-assign": "error",
```

Comment:
I prefer to to this from time to time, what is the reason to prevent this? Here I would even remove that check.

Reply:
[Here](https://eslint.org/docs/rules/no-multi-assign) are some examples, but I don't have a problem removing that check 🙂 I've removed it.

Reply:
I checked the eslint docs yesterday and I am still a bit a lazy guy. Still prefer to remove it. 

> I've removed it.

But you did not push the commit with the removal yet, right?

Reply:
I've updated the PR now.